### DISCUSSION TITLE
GitHub Advanced Security available with System license?

### DISCUSSION THREAD

**Discussion by ghost** [uillinois-community/uillinois-community.github.io#59] [Q&A] [answered]
Hi all, </br>Does the U of I System GitHub Enterprise license include any provisioning for [GitHub Advanced Security](https://docs.github.com/en/enterprise-cloud@latest/get-started/learning-about-github/about-github-advanced-security)? As far as I understand the [billing documentation](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-advanced-security/about-billing-for-github-advanced-security), it seems that the feature requires further licenses per committer for private repositories outside of what is provided by the System Enterprise license.

**Comment by edthedev** [comment, marked as answer]
@nsomers2 My understanding is that we do not have GitHub Advanced Security in our license. 

The team at [githubsupport@uillinois.edu](mailto:githubsupport@uillinois.edu) can answer more authoritatively.

If you know or are part of a team getting good use of the feature in public repositories, I would be glad to have a chat and we can start building a case for purchasing it. (I'm happy to add my voice to any conversation that adds security tooling for developers.)

**Reply by ghost** [reply]
Hi Ed, thanks for the info. I had asked out of curiosity while exploring GitHub's features for Enterprise customers, so no purchasing discussions are currently slated. It's good to know that you're willing to lend your voice if or when that time comes though.
